hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java

```diff
@@ -150,14 +149,16 @@
 import static org.apache.hadoop.ozone.om.OMConfigKeys.OZONE_OM_TRANSPORT_CLASS;
 import static org.apache.hadoop.ozone.om.exceptions.OMException.ResultCodes.TOKEN_EXPIRED;
 import static org.apache.hadoop.security.UserGroupInformation.AuthenticationMethod.KERBEROS;
+
+import org.apache.ozone.test.tag.Flaky;
 import org.apache.ratis.protocol.ClientId;
 import org.apache.ratis.util.ExitUtils;
 import org.bouncycastle.asn1.x500.RDN;
 import org.bouncycastle.asn1.x500.X500Name;
 import org.bouncycastle.asn1.x500.style.BCStyle;
 import org.bouncycastle.cert.X509CertificateHolder;
 import org.bouncycastle.cert.jcajce.JcaX509CertificateHolder;
-import org.junit.After;
+import org.junit.jupiter.api.AfterEach;
 import static org.junit.Assert.assertEquals;
```

Comment:
nit: these imports can be changed to Junit5.

Reply:
> these imports can be changed to Junit5

We usually skip that (for now).  The order of parameters has changed between 4 and 5 for some of the assert methods, so this can be done from an IDE that supports refactoring.  However, when performing that refactoring, it may create lines longer than 80 chars, for which we have a checkstyle rule.  Such lines have to be tweaked manually.

Reply:
I thought they were static imports and would be only import statement changes. I see your point of changing `org.junit.Assert` to `org.junit.jupiter.api.Assertions` and would create extra diff.